bootstrap/bootstrap.sh

```diff
@@ -0,0 +1,268 @@
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# See LICENSE file in the project root for full license information.
+
+# Note: This script should be compatible with the dash shell used in Ubuntu. So avoid bashisms! See https://wiki.ubuntu.com/DashAsBinSh for more info
```

Comment:
Removed this and added a shebang, this was just copied from the existing scripts.